packages/providers/src/operations.graphql

```diff
@@ -62,6 +62,8 @@ fragment receiptFragment on Receipt {
   reason
   ra
   rb
+  rc
+  rd
```

Comment:
Used down the chain over here:
https://github.com/FuelLabs/fuels-ts/blob/70f77ce4338da0a9ba7a2796b66486cd4efd1175/packages/providers/src/utils/receipts.ts#L146-L147